parkkihubi_hel/importers/permit_areas.py

```diff
@@ -15,20 +16,25 @@ class PermitAreaImporter(WfsImporter):
     """
     wfs_typename = 'Asukas_ja_yrityspysakointivyohykkeet_alue'
 
-    def import_permit_areas(self):
+    def import_permit_areas(self, permitted_user):
         permit_area_dicts = self.download_and_parse()
-        count = self._save_permit_areas(permit_area_dicts)
+        count = self._save_permit_areas(permit_area_dicts, permitted_user)
         logger.info('Created or updated {} permit areas'.format(count))
 
     @transaction.atomic
-    def _save_permit_areas(self, permit_areas_dict):
+    def _save_permit_areas(self, permit_areas_dict, permitted_user):
         logger.info('Saving permit areas.')
         count = 0
         permit_area_ids = []
         for area_dict in permit_areas_dict:
             permit_area, _ = PermitArea.objects.update_or_create(
                 domain=EnforcementDomain.get_default_domain(),
                 identifier=area_dict['identifier'],
+                permitted_user=get_user_model().objects.get_or_create(
```

Comment:
The area importer shouldn't create the user IMHO, so this should be just `.filter(username=...).get()`.